survey-sdk/src/test/java/co/nimblehq/survey/serializer/SerializerTest.kt

```diff
@@ -0,0 +1,150 @@
+package co.nimblehq.survey.serializer
+
+import co.nimblehq.survey.sdk.entity.AnswerEntity
+import co.nimblehq.survey.sdk.entity.MetaEntity
+import co.nimblehq.survey.sdk.entity.QuestionEntity
+import co.nimblehq.survey.sdk.entity.SurveyEntity
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import moe.banana.jsonapi2.ArrayDocument
+import moe.banana.jsonapi2.Document
+import moe.banana.jsonapi2.ObjectDocument
+import moe.banana.jsonapi2.ResourceAdapterFactory
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class SerializerTest {
+    lateinit var moshi: Moshi
+
+    @Before
+    fun setup() {
+        val jsonApiAdapterFactory: JsonAdapter.Factory = ResourceAdapterFactory.builder()
+            .add(SurveyEntity::class.java)
+            .add(QuestionEntity::class.java)
+            .add(AnswerEntity::class.java)
+            .build()
+        moshi = Moshi.Builder()
+            .add(jsonApiAdapterFactory) // ...
+            .add(KotlinJsonAdapterFactory())
+            .build()
+    }
+
+    @Test
+    fun `when parsing the right json string as list survey, it should return the SurveyListEntity`() {
+
+        val surveyList: ArrayDocument<SurveyEntity> =
+            moshi.adapter(Document::class.java)
+                .fromJson(JSON_SURVEY_LIST_TEST) as ArrayDocument<SurveyEntity>
+        assertTrue(surveyList.isNotEmpty())
+        //item test
```

Comment:
I'd recommend adding space after double slash `//` and the first character should be uppercase

https://nimblehq.co/compass/development/code-conventions/kotlin/#comments

Ex:
```
// Item test
```

Reply:
@hoangnguyen92dn Agree with this